cpu/stm32_common/periph/uart.c

```diff
@@ -154,6 +154,43 @@ int uart_init(uart_t uart, uint32_t baudrate, uart_rx_cb_t rx_cb, void *arg)
     return UART_OK;
 }
 
+#ifdef MODULE_PERIPH_UART_MODECFG
+int uart_mode(uart_t uart, uart_data_bits_t data_bits, uart_parity_t parity, uart_stop_bits_t stop_bits)
+{
+    assert(uart < UART_NUMOF);
+
+    uint32_t data_bits_holder = data_bits;
```

Comment:
Because of this error:

```
cpu/stm32f4/include/vendor/stm32f411xe.h:6695:39: error: large integer implicitly truncated to unsigned type [-Werror=overflow]
 #define USART_CR1_M_Msk               (0x1U << USART_CR1_M_Pos)                /*!< 0x00001000 */
                                       ^
cpu/stm32f4/include/vendor/stm32f411xe.h:6696:39: note: in expansion of macro 'USART_CR1_M_Msk'
 #define USART_CR1_M                   USART_CR1_M_Msk                          /*!<Word length                            */
                                       ^~~~~~~~~~~~~~~
cpu/stm32_common/periph/uart.c:163:17: note: in expansion of macro 'USART_CR1_M'
     data_bits = USART_CR1_M;
                 ^~~~~~~~~~~

```

Reply:
Another way to override would be to use `#define UART_INVALID_MODE 0xffffffff`. I'll test this variant.